RedHatInsights/drift-frontend
# [DRFT-325] Disable 'Add system' button with no baselinesWrite perms

To test:

- Go to line 29 in `src/App.js` and change to:
```
    const handlePermissionsUpdate = (hasCompareRead, hasBaselinesRead/*, hasBaselinesWrite*/, hasInventoryRead) => {
        setPermissions({
            hasCompareReadPermissions: hasCompareRead,
            hasBaselinesReadPermissions: hasBaselinesRead,
            hasBaselinesWritePermissions: /*hasBaselinesWrite*/false,
            hasInventoryReadPermissions: hasInventoryRead,
            arePermissionsLoaded: true
        });
    };
```
- Go to details for a baseline
- Select `Systems` tab

'Add system' button should be disabled. If you revert the above code then you'll see that the button is now enabled.

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
